src/annotate.ts

```diff
@@ -43,43 +43,38 @@ export default async function annotate({payload, _ajv = ajv, _schema = schema}:
     json: true
   };
 
-  let json: any = await _request(options)
+  const json: any = await _request(options)
     .then((res: any) => {
       return res;
     })
     .catch((err: any) => {
-      //console.error('_request error: ' + err);
       throw new Error('_request error: ' + err);
     });
 
-  json = json[0];
-  //console.log(JSON.stringify(json, undefined, 2));
-  // if there is no match, returns undefined
-  // if there are multiple matches, returns only the first one.. check this is appropriate.
+  // if there is no match, the request returns [undefined]
+  // if there are multiple matches, returns all.
 
-  if (json) {
+  if (json.length > 0 && typeof json[0] !== 'undefined') {
```

Comment:
@Istar-Eldritch I think this way checks correctly what I need.
if term = 'cancer', then json returned from request = [{results: etc} ] 
but
if term = 'zzzz', then json returned from request = [undefined]

Hence checking that json[0]  isn't undefined.

Reply:
The array has a string (`'undefined'`) as the first position?

Reply:
No when the term isn't found it returns 
```
[ undefined ] 
```

Reply:
Oh yeah, then you can just do `if (json[0])` but do it only if you are sure json is an array.